config.go

```diff
@@ -244,10 +244,14 @@ type Config struct {
 	// RequireNodeNames controls if the name of a node is required when sending
 	// a message to that node.
 	RequireNodeNames bool
+
 	// CIDRsAllowed If nil, allow any connection (default), otherwise specify all networks
 	// allowed to connect (you must specify IPv6/IPv4 separately)
 	// Using [] will block all connections.
 	CIDRsAllowed []net.IPNet
+
+	// MetricLabels is a map of optional labels to apply to all metrics emitted.
+	MetricLabels map[string]string
```

Comment:
Would it make sense to match the serf config and have this be a slice of metric labels?

Reply:
Either way looks good to me. If we switch to `MetricLabels []metrics.Label`, we can get rid of `mapToLabels` and pass []metrics.Label all the way from consul to memberlist, right?

Reply:
yep